pulp_python/app/models.py

```diff
@@ -33,6 +37,41 @@ class PythonDistribution(PublicationDistribution):
 
     TYPE = 'python'
 
+    def content_handler(self, path):
+        """
```

Comment:
@dkliban Someone more familiar with the content app / live APIs should take a look at this also

Reply:
This is the correct use of the content_handler() interface. I read about how PurePath.match() works and this looks like a good use of it.

Reply:
actually i was wondering, why the `content_handler` wasn't using an `async` interface, but it is declared like that in pulpcore. Not much you can do about it here.